api/Cargo.toml

```diff
@@ -37,7 +38,7 @@ reqwest = { version = "0.11", features = ["json"] }
 rocket = { version = "0.5.0-rc.3", features = ["json"] }
 schemars = "0.8"
 secstr = { version = "0.5", features = ["serde"] }
-serde = "1.0"
+serde = "1.0.188"
```

Comment:
It is not just that `bytesize` shouldn't carry any patch number. As [stated in the other reviews](https://github.com/aixigo/PREvant/pull/135#discussion_r1315686976) no dependency should have a patch number. If there is a reason for it, please, document it here.

Check **all** changes within this file.